source/features/search-releases.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {isReleasesOrTags} from '../libs/page-detect';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!isReleasesOrTags()) {
+		return false;
+	}
+
+	if (!isOnReleasePage()) {
+		return false;
+	}
+
+	addSelectMenu();
+	registerEventListener();
+}
+
+const isOnReleasePage = () => select.exists('.release-entry') || select.exists('.release');
+
+const getSelectedTagFromUrl = () => {
+	const result = location.href.match(/releases\/tag\/([^>/]*)/) || [];
+
+	return (result && result.length > 0) ? result[1] : null;
```

Comment:
Please use the suggestion I gave before (restructuring directly on the `const` so you don’t have to check for the length.

Reply:
TIL, location.href.match(/(?<=\/releases\/tag\/)([^\/]+)/).